src/ape_compile/_cli.py

```diff
@@ -62,3 +94,8 @@ def cli(contracts, compile_all, display_size):
             click.echo(f"  {name:<{indent}}  -  {size:>6,}B  ({pct:.2%})")
 
         click.echo()
+
+    # TODO: Remove this once compiling process works correctly
+    else:
+        # HACK: This force compiles all contracts
+        project.contracts
```

Comment:
What does "correctly" mean? Maybe we want to add a `project.compile()` method so that we can get rid of the `HACK` comment and not rely on the side effect of looking up the property.

Reply:
Definitely agreed. I didn't get to updating this for your caching PR yet